hummingbot/client/command/history_command.py

```diff
@@ -53,6 +53,30 @@ def history(self,  # type: HummingbotApplication
             if self.strategy_name != "celo_arb":
                 safe_ensure_future(self.history_report(start_time, trades, precision))
 
+    def get_history_trades_json(self,  # type: HummingbotApplication
+                                days: float = 0):
+        if self.strategy_file_name is None:
+            return
+        start_time = get_timestamp(days) if days > 0 else self.init_time
+        with self.trade_fill_db.get_new_session() as session:
+            trades: List[TradeFill] = self._get_trades_from_session(
+                int(start_time * 1e3),
+                session=session,
+                config_file_path=self.strategy_file_name)
+            return list([TradeFill.to_bounty_api_json(t) for t in trades])
+
+    def get_history_trades(self,  # type: HummingbotApplication
+                           days: float = 0):
+        if self.strategy_file_name is None:
+            return
+        start_time = get_timestamp(days) if days > 0 else self.init_time
+        with self.trade_fill_db.get_new_session() as session:
+            trades: List[TradeFill] = self._get_trades_from_session(
+                int(start_time * 1e3),
+                session=session,
+                config_file_path=self.strategy_file_name)
+            return TradeFill.to_pandas(trades)
+
```

Comment:
I didn't realise this method was a new one with the PR and since we've now replaced it with `get_history_trades_json` I think we should remove it :)